codeview/src/main/java/io/github/kbiakov/codeview/CodeView.kt

```diff
@@ -34,305 +27,147 @@ import java.util.*
  *
  * @author Kirill Biakov
  */
-class CodeView : RelativeLayout {
+open class CodeView : RelativeLayout {
```

Comment:
You are used Kotlin ```open class```, but I not found related ```open``` methods. May be this is not necessary in this case. Can you please explain your choice?